apps/judicial-system/backend/src/app/modules/case/filters/cases.filter.ts

```diff
@@ -252,6 +253,21 @@ const getDefenceUserCasesQueryFilter = (user: User): WhereOptions => {
                     },
                   ],
                 },
+                {
+                  [Op.and]: [
+                    { state: [CaseState.SUBMITTED, CaseState.RECEIVED] },
+                    {
+                      id: {
+                        [Op.in]: Sequelize.literal(`
```

Comment:
I don't understand why this doesn't work when fetching the case file list. Maybe there something else that I am missing 🤔 I have tested the query locally, and it works in postico but not in the app. It doesn't return a case which should be included on the case list. The only difference I spot in the console is the PII encoding: 
<img width="1180" alt="Screenshot 2025-04-14 at 15 39 24" src="https://github.com/user-attachments/assets/4f20c65f-c7af-4fcc-bd93-b8a46a2bfa07" />

Reply:
SOLVED